pkg/cleaner/cleaning.go

```diff
@@ -105,15 +109,8 @@ func (c *FileContentObfuscator) ObfuscateFile(inputFile string, outputFile strin
 	// we need to assess whether the file exists already to ensure we don't overwrite existing obfuscated data.
 	// that can happen while obfuscating file names and their paths.
 	// Additionally, the stat check is required because os.O_CREATE will implicitly os.O_TRUNC if a file already exist
```

Comment:
last nitpick for today: the comments above now should also go inside the method :)

Reply:
Done